src/hooks/query/useSendUserProfile.tsx

```diff
@@ -74,6 +74,20 @@ export const useSendUserProfile = () => {
   const queryClient = useQueryClient()
   const { signMessage } = useSignMessage()
 
+  const uploadImageMutation = useMutation({
+    mutationFn: (imageInfo: Blob) => {
+      const formData = new FormData()
+      formData.append('image', imageInfo)
+      return fetch(`/api/profile/${id}/upload-img`, {
+        method: 'PUT',
+        body: formData
+      })
+    },
+    onError: error => {
+      console.log(error)
+    }
```

Comment:
If you're doing nothing with the error result, it's probably better to not log it, since it may leak some undesired data

```suggestion
```